Gemfile

```diff
@@ -32,6 +32,7 @@ gem 'sass-rails'
 gem 'coffee-rails'
 gem 'activesupport-json_encoder'
 gem 'webpacker', git: 'https://github.com/rails/webpacker.git', branch: '3-x-stable'
+gem 'rb-readline'
```

Comment:
On my personal Mac I couldn't setup the project w/o. We can remove this for now I guess